esmvaltool/preprocessor/_mask.py

```diff
@@ -129,6 +161,46 @@ def mask_landsea(cube, fx_files, mask_out):
     return cube
 
 
+def mask_landseaice(cube, fx_files, mask_out):
+    """
+    Mask out either landsea (combined) or ice
+
+    Function that masks out either landsea (land and seas) or ice (Antarctica
+    and Greenland and some wee glaciers). It uses dedicated fx files (sftgif).
+
+    Args:
+    -----
+
+    * cube (iris.Cube.cube instance):
+        data cube to be masked.
+
+    * fx_files (list):
```

Comment:
Can there be more than one fx_file per model? if not, it probably makes more sense to just use the one filename and maybe rename it sftgif_file or something, because that is the only file allowed.

Reply:
@bouweandela that's a £500 question - I'd leave it as a list just now, only Lord Vader knows how many these bloody fx files can be, just in case, let's see what @mattiarighi has to say about it

Reply:
Since the fx_files are time-independent there should be only 1 file per fx-variable and per model.